projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,224 @@
+<ng-container *ngIf="replenishmentOrders$ | async as replenishmentOrders">
+  <div class="container">
+    <!-- HEADER -->
+    <div class="cx-replenishment-order-history-header">
+      <h3>
+        {{ 'orderHistory.replenishmentOrderHistory' | cxTranslate }}
+      </h3>
+    </div>
+
+    <!-- BODY -->
+    <div class="cx-replenishment-order-history-body">
+      <ng-container
+        *ngIf="replenishmentOrders.pagination.totalResults > 0; else noOrder"
+      >
+        <!-- Select Form and Pagination Top -->
+        <div class="cx-replenishment-order-history-sort top row">
+          <div
+            class="cx-replenishment-order-history-form-group form-group col-sm-12 col-md-4 col-lg-4"
+          >
+            <cx-sorting
+              [sortOptions]="replenishmentOrders.sorts"
+              [sortLabels]="getSortLabels() | async"
+              (sortListEvent)="changeSortCode($event)"
+              [selectedOption]="replenishmentOrders.pagination.sort"
+              placeholder="{{ 'orderHistory.sortByMostRecent' | cxTranslate }}"
+            ></cx-sorting>
+          </div>
+          <div
+            class="cx-replenishment-order-history-pagination"
+            *ngIf="replenishmentOrders.pagination.totalPages > 1"
+          >
+            <cx-pagination
+              [pagination]="replenishmentOrders.pagination"
+              (viewPageEvent)="pageChange($event)"
+            ></cx-pagination>
+          </div>
+        </div>
+        <!-- TABLE -->
+        <table class="table cx-replenishment-order-history-table">
+          <thead class="cx-replenishment-order-history-thead-mobile">
+            <th scope="col">
+              <span class="cx-replenishment-order-history-ellipses">
+                {{ 'orderHistory.replenishmentOrderId' | cxTranslate }}
+              </span>
+            </th>
+            <th scope="col">
+              {{ 'orderHistory.purchaseOrderNumber' | cxTranslate }}
+            </th>
+            <th scope="col">
+              <span class="cx-replenishment-order-history-ellipses">
```

Comment:
do we need this here? I thought it was only in a selected few which text was too long